README.md

```diff
@@ -16,7 +16,7 @@ Pipe a target image's Docker-inspected output into container's stdin
 
 ##### Failing Run, using default policy
 ```
-docker inspect 359039b8c10c | node index.js
+docker inspect 359039b8c10c | docker run -i bryanlatten/docker-image-policy
```

Comment:
just a quick consistency fix -- other examples are using `docker`, not `node` directly